### ISSUE TITLE
spelling mistake in example/ppi

### ISSUE THREAD

**Issue by Sanchez2020** [rusty1s/pytorch_geometric#379]
## 🐛 Bug
I found a **spelling mistake** in `example/ppi.py`.

Position on line **12** `val_dataset = PPI(path, split='test')` should be `val_dataset = PPI(path, split='val')`

## Additional context
I found that you only print out the F1 score of the validation set. I suggest also outputting the F1 score of the test set. Below is my code：
```python
for epoch in range(1, 101):
    loss = train()
    val_acc = test(val_loader)
    test_acc = test(test_loader)
    print('Epoch: {:02d}, train_loss: {:.4f}, val_F1: {:.4f}, test_F1: {:.4f}'.format(epoch, loss, val_acc, test_acc))

**Comment by rusty1s** [comment]
Ups, thank you. Fixed in master.
